src/descriptors/analysis.rs

```diff
@@ -593,15 +602,23 @@ impl LianaPolicy {
         // primary path with at least one key, and at least one timelocked recovery path with at
         // least one key.
         let subs = match policy {
-            SemanticPolicy::Threshold(1, subs) => Some(subs),
-            _ => None,
-        }
-        .ok_or(LianaPolicyError::IncompatibleDesc)?;
+            SemanticPolicy::Thresh(thresh) if thresh.is_or() && thresh.n() > 1 => {
```

Comment:
shouldn't we check thresh.k() == 1?
```suggestion
            SemanticPolicy::Thresh(thresh) if thresh.is_or() && thresh.n() > 1 && threshk() == 1 => {
```

Reply:
I think this is already checked by `thresh.is_or()`.

Reply:
I think it makes sense to keep this `thresh.n() > 1` condition here, but perhaps this is not strictly necessary as the previous version didn't have  `subs.len() > 1` and we already ensure further below that the primary path and recovery paths are not empty.